hw2/04-api.js

```diff
@@ -1,7 +1,29 @@
 /** Exercise 04 - API **/
 
-const url = 'https://restcountries.com/v3.1/all';
+const url = "https://restcountries.com/v2/all";
 
-// Add your code here
+const getData = () => {
 
-getData(url);
+    const cList = document.getElementById("results"); //get the item id from html
+    fetch(url)
+      .then((data) => data.json())
+      .then((data) => {
```

Comment:
Output is correct and fast. Numbers are formatted using `toLocaleString()` which I didn't even realize you could use without an argument. When you use it this way does it just get the locale of the system? If so, that would be super convenient. I want to go remove the argument from my version now haha. Looks good to me!